config/webpack.config.dev.js

```diff
@@ -8,6 +8,14 @@ const pkg = require(path.resolve(__dirname, '../package.json'))
 module.exports = {
   devtool: 'cheap-module-source-map',
   externals: ['cozy'],
+  module: {
+    rules: [
+      {
+        test: require.resolve('cozy-bar/dist/cozy-bar.js'),
```

Comment:
With that, when you will build cozy-bar it will need `cozy-bar/dist/cozy-bar.js` which is not available when you link the cozy-bar and run `yarn watch:mobile`